packages/back-end/src/infra/stacks/aws-healthomics-nested-stack.ts

```diff
@@ -23,122 +19,88 @@ export class AwsHealthOmicsNestedStack extends NestedStack {
     });
     this.setupIamPolicies();
 
-    this.dynamoDB = new DynamoConstruct(this, `${this.props.constructNamespace}-dynamodb`, {
-      devEnv: this.props.devEnv,
-    });
-    this.setupDynamoDBTables();
-
-    this.lambda = new LambdaConstruct(this, `${this.props.constructNamespace}`, {
-      ...this.props,
-      iamPolicyStatements: this.iam.policyStatements, // Pass declared Auth IAM policies for attaching to respective Lambda function
-      lambdaFunctionsDir: 'src/app/controllers/aws-healthomics',
-      lambdaFunctionsNamespace: `${this.props.constructNamespace}`,
-      lambdaFunctionsResources: {}, // Used for setting specific resources for a given Lambda function (e.g. environment settings, trigger events)
-      environment: {
-        // Defines the common environment settings for all lambda functions
-        ACCOUNT_ID: this.props.env.account!,
-        REGION: this.props.env.region!,
-        DOMAIN_NAME: this.props.appDomainName,
-        NAME_PREFIX: this.props.namePrefix,
-      },
-    });
+    // this.lambda = new LambdaConstruct(this, `${this.props.constructNamespace}`, {
+    //   ...this.props,
+    //   iamPolicyStatements: this.iam.policyStatements, // Pass declared Auth IAM policies for attaching to respective Lambda function
+    //   lambdaFunctionsDir: 'src/app/controllers/aws-healthomics',
+    //   lambdaFunctionsNamespace: `${this.props.constructNamespace}`,
+    //   lambdaFunctionsResources: {}, // Used for setting specific resources for a given Lambda function (e.g. environment settings, trigger events)
+    //   environment: {
+    //     // Defines the common environment settings for all lambda functions
+    //     ACCOUNT_ID: this.props.env.account!,
+    //     REGION: this.props.env.region!,
+    //     DOMAIN_NAME: this.props.appDomainName,
+    //     NAME_PREFIX: this.props.namePrefix,
+    //   },
+    // });
```

Comment:
is this code being kept for a particular reason or can it be deleted?

Reply:
It's going to be re-enabled once work on EG-870 starts.

The Lambda registration runs into an error if there are no lambda.ts files found by the stack.